unsloth/scripts/run-unsloth.sh

```
@@ -215,6 +216,9 @@ declare -a DATA_DESCRIPTORS=()
 declare -a SUGGEST_ENV=()
 declare -a SUGGEST_ARGS=()
 
+# Will hold optional descriptor for packaged spctl+config bundle
+SPCTL_PACKAGE_DESCRIPTOR=""
```

Shellcheck (0.11.0)

[warning] SPCTL_PACKAGE_DESCRIPTOR appears unused. Verify use (or export if used externally).

(SC2034)